src/hotspot/share/runtime/vmThread.cpp

```diff
@@ -136,6 +137,9 @@ void VMThread::create() {
     _perf_accumulated_vm_operation_time =
                  PerfDataManager::create_counter(SUN_THREADS, "vmOperationTime",
                                                  PerfData::U_Ticks, CHECK);
+    _perf_vm_thread_cpu_time =
+                 PerfDataManager::create_variable(NULL_NS, "vm_thread_time",
```

Comment:
Please move to the corresponding namespace and rename to something like `*.cpu_time`.